src/theme/components/product/ProductDetailBuyBox.vue

```diff
@@ -1,6 +1,9 @@
 <template>
     <div class="card bg-base-100 shadow-xl">
         <div class="card-body">
+            <button class="btn btn-ghost absolute top-6 right-8" @click="toggleWishlist">
+                <HeartIcon class="h-5 w-5" :class="{'fill-green-100': isOnWishlist}" />
```

Comment:
we should use a more "themeable" color e.g. fill-primary, fill-accent or something